container-images/llama-stack/Containerfile

```
@@ -0,0 +1,6 @@
+FROM registry.fedoraproject.org/fedora:42
+
+RUN dnf -y install pip && \
+    pip install -U llama-stack llama_stack-client --prefix=/usr
```

```suggestion
    pip install -U llama-stack --prefix=/usr
```

`llama-stack-client` is a dependency of `llama-stack`: https://github.com/meta-llama/llama-stack/blob/main/requirements.txt#L24